app/invocation/invocation_action_card.tsx

```diff
@@ -315,14 +276,10 @@ export default class InvocationActionCardComponent extends React.Component<Props
   }
 
   handleFileClicked(node: InputNode) {
-    let digestString = node.obj.digest?.hash + "/" + node.obj.digest?.sizeBytes;
-    let dirUrl =
-      "bytestream://" +
-      this.props.model.getCacheAddress() +
-      "/blobs/" +
-      this.props.model.getDigestFunctionDir() +
-      digestString;
+    if (!node.obj?.digest) return;
 
+    let dirUrl = this.props.model.getBytestreamURL(node.obj.digest);
```

Comment:
super nit: `const`?

Reply:
I usually use `const` for new code but since the benefits are pretty minor I prefer to stick with whatever the current code is doing to avoid adding extra noise to the diff.